src/Http/Controllers/VoyagerBreadController.php

```diff
@@ -423,7 +423,9 @@ protected function cleanup($dataType, $data)
     public function deleteBreadImages($data, $rows)
     {
         foreach ($rows as $row) {
-            $this->deleteFileIfExists($data->{$row->field});
+            if ($data->{$row->field} != 'users/default.png') {
```

Comment:
As @emptynick suggested, `'users/default.png'` should not be hardcoded, and should be replaced with the config var.